### ISSUE TITLE
Errors: Might want to rethink colours used

### ISSUE THREAD

**Issue by ArakTaiRoth** [GovAlta/ui-components#2364]
# Info

This issue is raised by the UAIM Accessibility Report. This is a Level 2 issue. "Find red really difficult in any situation. Don't put text in red!". The recommendation here is not to remove red from error text, but don't use red outside of error text. But the other recommendation is to change the red colour used to #D32F2F.

## Acceptance Criteria:

1. Change error design tokens to #D32F2F

**Comment by Spark450** [comment]
The suggested "red" does not meet the AA contrast ratio guideline when used as text on Grey 100, which is currently the background of our non-interactive container. I believe the suggested "red" should be adjusted to meet the AA contrast ratio requirement for normal text.

**Comment by ArakTaiRoth** [comment]
@twjeffery Look into this for an actual colour to change to, if we even want to

**Comment by twjeffery** [comment]
The suggested colour also doesn't make a big change to the colour contrast. 

Current error colour contrast: **4.58:1**
Suggested colour contrast: **4.98:1**

<img width="558" alt="Image" src="https://github.com/user-attachments/assets/8a544c66-32ff-4b19-8226-e29f6e843a0c" />

On greyscale 100, both don't meet accessibility of 4.5:1. (but the suggested colour comes close). As @Spark450 suggests above, when we change this colour we should ensure we meet accessibility contrast ratios on all contexts (slightly darker).

<img width="306" alt="Image" src="https://github.com/user-attachments/assets/c727660d-5841-4fe7-95c2-86242f0967c7" />

**Comment by ArakTaiRoth** [comment]
Closing this as it isn't relevant anymore with our current redesign
